modules/docs/mdx/10.0-UPGRADE_GUIDE.mdx

```diff
@@ -70,6 +71,12 @@ Preview and the `Menu` in Main.
 
 ## Deprecations
 
+### Input Icon Container
+
+We've deprecated `InputIconContainer` from `TextInput` component, because it doesn't handle
+bidirectionality or icons at the start of an input. `InputGroup` should be used instead of
+deprecated `InputIconContainer`.
```

Comment:
If we have a story for `InputGroup` we should point to it

Reply:
I don't see there is any specific story `InputGroup`, but it's described in `Icons` story, so I add link to it

Reply:
Ah good point! Maybe we need to update this at some point!